### PR TITLE
added cloudhosted instructions

### PR DESCRIPTION
**Pull request by tt-le**
Todo:
- [x] Add cd to start command

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -237,8 +264,21 @@ property for all the systems:
 }
 ----
 
+// static guide instructions:
+ifndef::cloud-hosted[]
 You can revisit the http://localhost:9085/inventory/systems[^] URL after a while, and you will notice the CPU `systemLoad`
```
Comment (dmuelle) [inline README.adoc:269]: You can revisit the http://localhost:9085/inventory/systems[^] URL after a while, and you will notice the CPU `systemLoad`
--->
You can revisit the http://localhost:9085/inventory/systems[^] URL after a while and the CPU `systemLoad` property for the systems  is changed.

this sentence is a little vague? is it possible to clarify how it has changed, or what about it is different? What is the user seeing that is different?

**Review by dmuelle — CHANGES_REQUESTED** [review]
A question for clarification

File: README.adoc (hunk 1 above)
Comment (gkwan-ibm) [reply README.adoc:269]: hi @dmuelle
The value of the systemLoad property is changed. How's the following?
--->
You can revisit the http://localhost:9085/inventory/systems[^] URL after a while and the value of `systemLoad` property for the system is changed.

Comment (dmuelle) [reply README.adoc:269]: That's good, I would only add "the", and retain the plural of "systems", if the user is looking at values for multiple instances of the property:

You can revisit the http://localhost:9085/inventory/systems[^] URL after a while and the value of the `systemLoad` property for the systems is changed.

File: README.adoc [hunk 2]
```diff
@@ -237,8 +264,21 @@ property for all the systems:
...
 property for the systems changed.
+endif::[]
+
+// cloud-hosted guide instructions:
+ifdef::cloud-hosted[]
+You can revisit the **inventory** service after a while by running the following curl command, and you will notice the CPU **systemLoad**
```
Comment (dmuelle) [inline README.adoc:275]: based on previous comment, recommend here:

You can revisit the **inventory** service after a while by running the following curl command:
```
curl http://localhost:9085/inventory/systems
```
Notice the CPU **systemLoad** property for the systems is changed.

**Review by dmuelle — CHANGES_REQUESTED** [review]

**Comment by tt-le** [comment]
@dmuelle thank you for the feedback! I made both changes. Was there anything else you wanted to add?

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**